plugins/frontend/backstage-plugin-github-pull-requests/src/components/Home/YourOpenPullRequestsCard/Content.tsx

```diff
@@ -26,10 +26,15 @@ import {
 } from '../../useGithubLoggedIn';
 import Alert from '@material-ui/lab/Alert';
 
-const OpenPullRequestsContent = () => {
-  const { loading, error, value } = useGithubSearchPullRequest(
-    `is:open is:pr author:@me archived:false`,
-  );
+type OpenPullRequestsCardProps = {
+  query: string;
+};
+
+const defaultPullRequestsQuery = "is:open is:pr author:@me archived:false"
+
+const OpenPullRequestsContent = (props: OpenPullRequestsCardProps) => {
+  const { query = defaultPullRequestsQuery } = props;
```

Comment:
TIL you can do this! I think it looks pretty clear 🙂

Reply:
oh yea! Indeed it is clean imo!